src/components/home_page.js

```diff
@@ -55,36 +50,37 @@ const ControlledTabs = React.createClass({
   },
 
   handleSelect(key) {
-    {/*alert('selected ' + key);*/}
     this.setState({key});
   },
 
   render() {
     return (
-      <Tabs activeKey={this.state.key} onSelect={this.handleSelect} id="controlled-tab-example">
+      <Tabs activeKey={this.state.key} onSelect={this.handleSelect} id="Controlled-Tabs">
         <Tab eventKey={1} title="My Bug Profiles"><MyBugProfiles /></Tab>
         <Tab eventKey={2} title="My Projects"><MyProjects /></Tab>
       </Tabs>
     );
   }
 });
 
-var MyBugProfiles = React.createClass({
-  render: function(){
-    return(
-      <div>
-      <br/>
-        Lorem ipsum dolor sit amet, consectetur adipiscing elit.
-        Curabitur vitae dignissim leo. Donec rhoncus iaculis erat sit amet molestie. Quisque vitae vestibulum lacus, ut aliquet sapien.
-        Praesent id tortor facilisis, fermentum neque id, luctus eros.
-        Nulla sapien urna, euismod sit amet faucibus a, ultricies at neque.
-        Morbi id justo vel erat pretium aliquet in vel ipsum. In eu mauris at tortor cursus gravida nec sit amet odio.
-
-      </div>
-  );
-  }
-});
+class MyBugProfiles extends React.Component {
+    render(){
+        return(
+            <div>
+                <br/>
+                <p>
+                Lorem ipsum dolor sit amet, consectetur adipiscing elit.
+                Curabitur vitae dignissim leo. Donec rhoncus iaculis erat sit amet molestie. Quisque vitae vestibulum lacus, ut aliquet sapien.
+                Praesent id tortor facilisis, fermentum neque id, luctus eros.
+                Nulla sapien urna, euismod sit amet faucibus a, ultricies at neque.
+                Morbi id justo vel erat pretium aliquet in vel ipsum. In eu mauris at tortor cursus gravida nec sit amet odio.
+                </p>
+            </div>
+        );
+    }
+}
 
+<<<<<<< HEAD
```

Comment:
Looks like a merge conflict was committed. @ctshudy can you delete lines 83-99 and line 117?